node/src/tasks/bsp_submit_proof.rs

```diff
@@ -329,7 +284,7 @@ where
 
     async fn generate_key_proof(
         &self,
-        file_key: HasherOutT<T>,
+        file_key: H256,
```

Comment:
How about using `HasherOutT<StorageProofsMerkleTrieLayout>` in this document, instead of `H256`?